Documentation/block.md

```diff
@@ -99,3 +105,49 @@ kubectl delete -f mysql.yaml
 kubectl delete -n rook-ceph pool replicapool
 kubectl delete storageclass rook-ceph-block
 ```
+
+## Advanced Example: Erasure Coded Block Storage
+
+If you want to use erasure coded pool with RBD, you need to create two pools (as seen below): one erasure coded and one replicated.
+You need to specify the replicated pool as the `pool` parameter. It is used for the metadata of the images.
+The erasure coded pool must be set as the `dataPool` parameter below. It is used for the data of the images.
+
+```yaml
+apiVersion: ceph.rook.io/v1beta1
+kind: Pool
+metadata:
+  name: replicated-metadata-pool
+  namespace: rook-ceph
+spec:
+  replicated:
+    size: 3
+---
+apiVersion: ceph.rook.io/v1beta1
+kind: Pool
+metadata:
+  name: ec-data-pool
+  namespace: rook-ceph
+spec:
+  # Make sure you have enough OSDs to support the replica size or erasure code chunks.
+  # For the below settings, you need at least 3 OSDs.
```

Comment:
Two remarks here:
- maybe also assert here that erasure coded pools need all OSDs to be bluestore?
- the default pool `failureDomain` is `host`, so you actually need 3 hosts with one or more OSDs each

Reply:
@lbogdan I have implemented those points, what do you think about it now?